### PR TITLE
VIH-7891 enable service web for mobile tablet

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-7891

### Change description ###
- enabling service web for android/tablets

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.release.yml [hunk 1]
```diff
@@ -290,6 +306,8 @@ stages:
                 value: $(service_web_url_staging)
               active:
                 value: $(service_web_url)
+          - name: VhServices:EnableMobileSupport
```
Comment (PrasannaDommalapati) [inline azure-pipelines.release.yml:309]: this key is not added in azurepipelines.yml and .nightly.yml

Comment (omio-ma) [reply azure-pipelines.release.yml:309]: can we set that to false by default in app config?

**Pull request closed without merging**